libraries/SD_MMC/src/SD_MMC.cpp

```diff
@@ -186,6 +203,19 @@ bool SDMMCFS::begin(const char *mountpoint, bool mode1bit, bool format_if_mount_
   }
   _mode1bit = mode1bit;
 
+#ifdef SOC_SDMMC_IO_POWER_EXTERNAL
+  sd_pwr_ctrl_ldo_config_t ldo_config = {
+    .ldo_chan_id = _pin_power,
```

Comment:
Are you not by any chance confusing GPIO number with LDO channel number here?

Reply:
Yeah I get it wrong here. 
Should it be always channel 4 for P4 or should I let the user to specify?

Reply:
This depends on the development board. ESP32-P4 EV board powers IO lines of slot 0 using LDO channel 4. Other boards may use other LDO channels or even use an external LDO, or directly power them from VDD.

I think you need to define this in the board-specific header file.

Reply:
Solved :)